daphne/src/lib.rs

```diff
@@ -716,6 +716,15 @@ impl std::str::FromStr for VdafConfig {
     }
 }
 
+impl Display for VdafConfig {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            VdafConfig::Prio3(conf) => write!(f, "Prio3({conf})"),
```

Comment:
nit: For consistency with match on `VdafConfig` elsewhere 
```suggestion
            VdafConfig::Prio3(prio3_config) => write!(f, "Prio3({conf})"),
```